db/seed.sql

```diff
@@ -757,8 +757,8 @@ INSERT INTO `catalog_eav_attribute` VALUES (127,NULL,1,0,0,0,0,0,0,0,0,1,0,'down
 INSERT INTO `catalog_eav_attribute` VALUES (128,NULL,0,1,0,0,0,0,0,0,0,1,0,NULL,0,0,0,0,0,0,0,0,1,NULL);
 INSERT INTO `catalog_eav_attribute` VALUES (129,NULL,2,1,1,0,0,0,0,0,0,1,0,'simple,virtual,bundle,downloadable,configurable',0,0,0,0,0,1,0,1,1,NULL);
 INSERT INTO `catalog_eav_attribute` VALUES (130,'Magento\\GiftMessage\\Block\\Adminhtml\\Product\\Helper\\Form\\Config',1,1,0,0,0,0,0,0,0,0,0,NULL,0,0,0,0,0,1,0,0,1,NULL);
-INSERT INTO `catalog_eav_attribute` VALUES (131,NULL,0,1,0,0,0,0,1,0,0,0,0,NULL,0,0,0,0,0,1,1,1,1,NULL);
-INSERT INTO `catalog_eav_attribute` VALUES (132,NULL,0,1,0,0,0,0,1,0,0,0,0,NULL,0,0,0,0,0,1,1,1,1,NULL);
+INSERT INTO `catalog_eav_attribute` VALUES (131,NULL,0,1,1,1,0,1,1,0,0,1,0,NULL,1,0,0,0,0,1,1,1,1,NULL);
+INSERT INTO `catalog_eav_attribute` VALUES (132,NULL,0,1,1,1,0,1,1,0,0,1,0,NULL,1,0,0,0,0,1,1,1,1,NULL);
```

Comment:
So `is_filterable_in_search` "Can be used only with catalog input type Dropdown, Multiple Select and Price" and is disabled for other input types.  In fact, `is_filterable` has the same rule, so setting it to `1` here makes no difference.

However you are right about `is_comparable`, it should be set to `1` so that the attribute is displayed on the Compare Products page.

Will fix both of these.